hr_employee_age/models/hr_employee.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2015 Salton Massally (<smassally@idtlabs.sl>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from dateutil.relativedelta import relativedelta
+
+from odoo import api, fields, models
+
+
+class HrEmployee(models.Model):
+    _inherit = "hr.employee"
+
+    age = fields.Integer(string="Age", readonly=True, compute="_compute_age")
```

Comment:
`string="Age"` and `readonly=True` are the default for this field:

```suggestion
    age = fields.Integer(compute="_compute_age")
```